GNUmakefile

```diff
@@ -40,28 +32,27 @@ test-compile:
 	fi
 	@echo "==> Compiling test binary..."
 	go test -c $(TEST) $(TESTARGS)
-
-.PHONY: build test testacc vet fmt fmtcheck errcheck lint test-compile init-git-hooks
```

Comment:
this was out of sync with available commands so I decided to define hem next to the command so we keep them updated